### DISCUSSION TITLE
Add capacity reservation into cloudformation template in aws

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3787] [Hubs Cloud Questions] [answered]
Hello, 

Is there a way to add capacity reservation in the template in cloudformation in aws  ? 
In my zone, There is no c5.large or c5.xlarge avalaible and I think that we can add capacity reservation , but I don't know if this fonctionnality is add into the hubs template ? 

thank you for your answer 

Marc

**Comment by robinkwilson** [comment, marked as answer]
Thanks for bringing that to our attention. 

Do you know if the c5.large or c5.xlarge servers are not available in your region at all? OR are they currently busy being used by other folks.
What region are you in?

**Reply by robinkwilson** [reply]
Marking as answered for now, please unmark as answered when you include more information!!
